swagger_py_codegen/templates/flask/validators.tpl

```diff
@@ -85,7 +85,7 @@ def request_validate(view):
             validator = FlaskValidatorAdaptor(schema)
             result, errors = validator.validate(value)
             if errors:
-                abort(422, message='Unprocessable Entity', errors=errors)
+                abort(400, message='Invalid Request', errors=errors)
```

Comment:
in my case, status code 422 will trigger another error, which is weird. 

i will do more test upon this.